extensions/grunt/src/main.ts

```diff
@@ -70,12 +70,32 @@ interface GruntTaskDefinition extends vscode.TaskDefinition {
 	file?: string;
 }
 
+async function findGruntCommand(rootPath: string): Promise<string> {
+	let gruntCommand: string;
+	let platform = process.platform;
+	if (platform === 'win32' && await exists(path.join(rootPath, 'node_modules', '.bin', 'grunt.cmd'))) {
```

Comment:
I think this part can use the same code in line 150 instead of just copy from b6089b1db51dbfb7b096a829261a0b3b9370d6e5. 
The code in line 150 is duplicated with here. You can see b6089b1db51dbfb7b096a829261a0b3b9370d6e5 also move the code to here.

Reply:
yeah, you're right about that

Reply:
Im going too fixed it soon